### PR TITLE
Update Q# notebooks to 0.15 syntax.

### PR DESCRIPTION
**Pull request by cgranade**

### CODE REVIEW

File: samples/algorithms/database-search/Database Search.ipynb [hunk 1]
```diff
@@ -326,7 +330,7 @@
    "file_extension": ".qs",
    "mimetype": "text/x-qsharp",
    "name": "qsharp",
-   "version": "0.10"
+   "version": "0.14"
```
Comment (msoeken) [inline samples/algorithms/database-search/Database Search.ipynb:333]: What is this version number?

**Review by msoeken — APPROVED** [review]
Looks good to me, thanks!

File: samples/algorithms/database-search/Database Search.ipynb (hunk 1 above)
Comment (guenp) [reply samples/algorithms/database-search/Database Search.ipynb:333]: @msoeken I think this is metadata generated by Jupyter.

Comment (cgranade) [reply samples/algorithms/database-search/Database Search.ipynb:333]: Indeed, that metadata is generated from here: https://github.com/microsoft/iqsharp/blob/406629f9378aebefd1cd54e5877471c8d61cfbc6/src/Kernel/KernelProperties.cs#L28

That needs manually updated with version bumps, but that got missed during the 0.14 → 0.15 bump.

**Pull request merged**